### PR TITLE
GPC-NONE: Add powertools tracing

### PR DESCRIPTION
**Pull request by oskarwilliams**

### CODE REVIEW

File: lib/build.gradle [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
     id "jacoco"
+    id "io.freefair.aspectj.post-compile-weaving" version "6.6.3"
 }
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
+
```
Comment (ethanmills) [inline lib/build.gradle:12]: whitespace?

File: lambdas/authorisation/src/main/java/uk/gov/di/data/lep/Authorisation.java [hunk 2]
```diff
@@ -2,10 +2,14 @@
 
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
+import software.amazon.lambda.powertools.logging.Logging;
+import software.amazon.lambda.powertools.tracing.Tracing;
 
-public class Authorisation implements RequestHandler {
+public class Authorisation implements RequestHandler<Object, Object> {
```
Comment (ethanmills) [inline lambdas/authorisation/src/main/java/uk/gov/di/data/lep/Authorisation.java:8]: Is this change needed? Can we be any more specific?

File: lib/build.gradle (hunk 1 above)
Comment (oskarwilliams) [reply lib/build.gradle:12]: sad

File: lambdas/authorisation/src/main/java/uk/gov/di/data/lep/Authorisation.java (hunk 2 above)
Comment (oskarwilliams) [reply lambdas/authorisation/src/main/java/uk/gov/di/data/lep/Authorisation.java:8]: This is to stop it shouting at me, but we don't need it (it is a minor tidy)

File: lib/build.gradle (hunk 1 above)
Comment (oskarwilliams) [reply lib/build.gradle:12]:
```suggestion
```
